### PR TITLE
Added missing recommendations command and cleaned up code

### PR DESCRIPTION
**Pull request by inclement**
Added this missing command, which existed in the argument parser but wasn't actually implemented.

Not sure what format we really want, but just having the command available is probably an improvement for now.

I thought about removing the argument entirely, but it does seem useful to be able to print the p4a recommendations easily.

### CODE REVIEW

**Comment by inclement** [comment]
Also updated minimum python version supported to 3.6, as originally agreed in https://github.com/kivy/python-for-android/pull/1918 - it's my mistake that I left it at 3.4 in that PR. This minimum version is now essential since I added f-strings to the code.

File: pythonforandroid/recommendations.py [hunk 1]
```diff
@@ -218,3 +220,14 @@ def check_python_version():
     ):
 
         raise BuildInterruptingException(PY_VERSION_ERROR_TEXT)
+
+def print_recommendations():
```
Comment (AndreMiras) [inline pythonforandroid/recommendations.py]: Linter unhappy :grimacing: 
```
pythonforandroid/recommendations.py:224:1: E302 expected 2 blank lines, found 1
```

File: pythonforandroid/recommendations.py [hunk 2]
```diff
@@ -187,7 +189,7 @@ def check_ndk_api(ndk_api, android_api):
 
 
 MIN_PYTHON_MAJOR_VERSION = 3
-MIN_PYTHON_MINOR_VERSION = 4
+MIN_PYTHON_MINOR_VERSION = 6
```
Comment (AndreMiras) [inline pythonforandroid/recommendations.py]: Nice nice :smile:

**Comment by inclement** [comment]
This one is also failing only on osx with the pyconfig.h error...not sure what's wrong, although it seems fairly clear this PR is not the culprit.

**Comment by inclement** [comment]
Tests failing only due to macos issue also present on trunk.

**Comment by inclement** [comment]
Thanks!
Looks like we'll have to do something else about this issue breaking the develop branch tests :/

**Comment by AndreMiras** [comment]
For records, this got fixed recently via https://github.com/kivy/python-for-android/pull/2063

**Pull request merged**